test/bigtest/network/scenarios/default.js

```diff
@@ -0,0 +1,5 @@
+/* istanbul ignore file */
```

Comment:
Should this be really ignored? I just wonder whether this should not be ignored already.

Reply:
Fixed